packages/styles/scss/components/content-switcher/_tokens.scss

```
@@ -0,0 +1,110 @@
+//
+// Copyright IBM Corp. 2020
```

https://github.com/carbon-design-system/carbon/issues/18954#issuecomment-2813348278

```suggestion
// Copyright IBM Corp. 2025
```

---

This comment was marked as resolved but I don't see a change. Was there supposed to be one?